### PR TITLE
Rootstock Flyover dApp

### PR DESCRIPTION
**Pull request by ahsan-javaiid**
### Devop

* \[X] Added entry to ./changelog
* \[ ] Add PR label

### Description
Added flyover dApp for Rootstock network based on flyover protocol. This app provides easier way of transfer of BTC between the Bitcoin mainchain and the Rootstock sidechain with RIF Flyover protocol within the wallet.

Flyover website: https://flyover.rifos.org/

### Demo

https://github.com/MyEtherWallet/MyEtherWallet/assets/104683677/8b86a4b8-8098-48d8-94ba-ae484729a6c1

https://github.com/MyEtherWallet/MyEtherWallet/assets/104683677/6383c1e0-663c-4229-acc3-5c81d9799986

### CODE REVIEW

**Comment by ahsan-javaiid** [comment]
Hi @gamalielhere

This pull request (Flyover dApp) is ready to merge. Apologies for taking so long on this. Flyover team updated the protocol and now it's ready to review and merge. 
The liquidity on flyover is limited so just bare with that (A user could try later if in case he face not enough liquidity notification).

Feel free to review and merge the pull request.

cc: @alepc253

**Review by gamalielhere — CHANGES_REQUESTED** [review]

File: src/dapps/flyover-dapp/components/pegout/GetQuote.vue [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+          this.$emit('onSubmit', {
+            ...quoteReply[0],
+            quoteUrl: this.provider,
+            siteKey: this.siteKey
+          });
+        }
+      } catch (e) {
+        let detail = '';
+        if (e.details && e.details.maxValueTotransfer) {
+          detail = e.details.maxValueTotransfer / 10 ** 18;
```
[the remark this answers is not in the document]
Comment (ahsan-javaiid) [reply src/dapps/flyover-dapp/components/pegout/GetQuote.vue]: Done!

File: src/dapps/flyover-dapp/components/pegout/GetQuote.vue [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  },
+  methods: {
+    async getQuote() {
+      this.loading = true;
+      this.msg = '';
+      const payload = {
+        bitcoinRefundAddress: this.btcRefund,
+        rskRefundAddress: this.rskAddress,
+        to: this.btcAddress,
+        valueToTransfer: parseFloat(this.amount) * 10 ** 18
```
[the remark this answers is not in the document]
Comment (ahsan-javaiid) [reply src/dapps/flyover-dapp/components/pegout/GetQuote.vue]: Done!

File: src/dapps/flyover-dapp/modules/ModulePegout.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  },
+  watch: {
+    onStep() {}
+  },
+  mounted() {},
+  methods: {
+    onSubmit(val) {
+      this.onStep += 1;
+      if (this.onStep === 2) {
+        this.quoteAmount = val.quote.value / 10 ** 18;
```
[the remark this answers is not in the document]
Comment (ahsan-javaiid) [reply src/dapps/flyover-dapp/modules/ModulePegout.vue]: Done!

File: src/dapps/flyover-dapp/components/pegout/DepositFunds.vue [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+export default {
+  name: 'DepositFunds',
+  components: {},
+  props: {
+    hash: {
+      default: '',
+      type: String
+    }
+  },
+  data() {
```
[the remark this answers is not in the document]
Comment (ahsan-javaiid) [reply src/dapps/flyover-dapp/components/pegout/DepositFunds.vue]: Done!

**Pull request merged**